hashin.py

```diff
@@ -75,6 +76,12 @@
     action='store_true',
     default=False,
 )
+parser.add_argument(
+    '--dry-run',
+    help='Don\'t touch requirements.txt and just show hashes',
```

Comment:
Nit: Most style guides allows for switching quotes to avoid escaping quotes in a string.

Reply:
I wouldn't worry about this one. Once this PR is squared away there are no open PRs and I can take a massive knife of black and therapist to this code base and then these kinds of nits become unimportant.